src/EditorFeatures/VisualBasicTest/SpellCheck/SpellCheckSpanTests.vb

```diff
@@ -0,0 +1,310 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports Microsoft.CodeAnalysis.Editor.UnitTests.SpellCheck
+Imports Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.SpellCheck
+    <UseExportProvider>
+    Public Class SpellCheckSpanTests
+        Inherits AbstractSpellCheckSpanTests
+
+        Protected Overrides Function CreateWorkspace(content As String) As TestWorkspace
+            Return TestWorkspace.CreateVisualBasic(content)
+        End Function
+
+        <Fact>
+        Public Async Function TestComment1() As Task
+            Await TestAsync("'{|Comment: Goo |}")
+        End Function
+
+        <Fact>
+        Public Async Function TestComment2() As Task
+            Await TestAsync("
+'{|Comment: Goo |}")
+        End Function
+
+        <Fact>
+        Public Async Function TestDocComment1() As Task
+            Await TestAsync("
+'''{|Comment:goo bar Comment:baz|}
```

Comment:
Is the second `Comment:` here intentional?

Reply:
it is not :)  nice catch.